data/txHandler_test.go

```diff
@@ -2774,6 +2774,8 @@ func TestTxHandlerValidateIncomingTxMessage(t *testing.T) {
 
 	handler, err := makeTestTxHandler(ledger, cfg)
 	require.NoError(t, err)
+	handler.Start()
+	defer handler.Stop()
```

Comment:
The handler variable gets set to a new test handler on line 2810 - which handler will this refer to when `stop()` is called (I would think latest referenced object).

Reply:
there is a new scope => new local variable